code/datums/actions/observer_action.dm

```diff
@@ -48,6 +48,11 @@
 
 /datum/action/observer_action/take_ssd_mob/action_activate()
 	var/mob/dead/observer/dead_owner = owner
+
+	if(!GLOB.ssd_posses_allowed)
```

Comment:
I feel like there's probably a better solution since this means you can't take ssd marines either but eh... I can't think of anything right now.

Reply:
Would require more changes to how mobs are added to the SSD list.